features/google/src/views/explore/explore.css

```diff
@@ -12,24 +10,6 @@
     overflow-x: hidden;
 }
 
-.explore-view .report-button {
```

Comment:
All this was subsumed into the poly-look component?

Reply:
yep :)